config/webpacker.yml

```diff
@@ -4,7 +4,7 @@ default: &default
   source_path: app/javascript
   source_entry_path: packs
   public_output_path: packs
-  cache_path: tmp/cache/webpacker
+  cache_path: node_modules/.cache
```

Comment:
ah, so webpacker will use cache `node_modules/.cache` but then I don't see where we cache this directory.

Reply:
I it will be included in the npm-cache, no?

Reply:
ah, got your idea. But this `.cache` is generated by webpacker, not node, correct? So node cache can restore something that is not what webpacker would generate. Or no?